lib/any_login.rb

```diff
@@ -24,6 +24,10 @@ module Provider
   mattr_accessor :klass_name
   @@klass_name = 'User'
```

Comment:
Does this PR break the ability to use the old `klass_name` style? If so it's likely going to require all users to update their configuration which is going to be a breaking change. Thoughts on just using `klass_name` and checking whether it is an array or not in the code and then handling it accordingly?

Reply:
This PR does not break `klass_name` style because the default value of `klass_names` is `[@@klass_name]`.
I think this is simpler than `klass_name`, which accepts both strings and arrays.

If you want to use any_login in multiple models, you can use `klass_names`, and if that is not necessary, you can continue to use `klass_name`.

Reply:
You can confirm that the `klass_name` style is not broken in demo apps.

- test/rails_app/authlogic
- test/rails_app/clearance

Reply:
I just did a test in the demo devise application and it seems to confirm that `klass_name` is currently broken. I switched the devise demo application to use `klass_name = 'Staff'` and I was only presented with users and not staff:


https://user-images.githubusercontent.com/2308869/209186910-aa2102a0-0b00-4e64-a87f-7fc59f16b38b.mov

The reason this is happening it the `klass_names` is always being initialized with the default value of `klass_name` (which is User) and it is never taking into account the users input. The easiest way to fix this would be to just use `klass_name` for everything and just handle it properly if `klass_name` is an array.

<img width="962" alt="Screenshot 2022-12-22 at 11 59 37 AM" src="https://user-images.githubusercontent.com/2308869/209187343-35320d67-ce26-4ead-87d4-3cd449372700.png">